docs/_style.scss

```diff
@@ -43,6 +43,20 @@ h3 {
 .slick-slider {
   margin: 30px auto 50px;
 }
+.slick-dots {
```

Comment:
The left margin comes from the default browser behavior when we add `<ul>` tags.